### ISSUE TITLE
Species availability by book is wrong in certain cases, causes issue with Talents

### ISSUE THREAD

**Issue by Felderburg** [bcholmes/StarTrek2d20#36]
So I was looking at the species that are available, and I noticed something odd. Some species are available when you select books that they aren't in, but they show up without talents.

The one specific example I can point to is Cardassians. When using Core only, or Core and Beta (e.g.), they can be selected as a species, but they don't have talents listed. However, when selecting Alpha, their talents show up.

There are others, but I didn't feel like doing an exhaustive search, especially since my usual use-case is 'select all' for books. I do have a table with the species and their source book listed here: https://airtable.com/shrevx2C19EPOQnl3 if that might be of use.

**Comment by bcholmes** [comment]
The codebase actually has a few different "versions" of some species (mostly adversaries), which reflect different ways those species have been represented. For example, Cardassians are described on page 329 of the Core rulebook. That "version" of Cardassians have the 3 Species Attribute bonuses spelled out, but no Talents are defined. There's some code in place to use the "better" version of the species when more sources are in place. 

So, in short, I think that it's behaving as designed.

**Comment by Felderburg** [comment]
That makes some sense, but I am under the impression that the tool gets updated as books come out. So my assumption is that the original creator (azjerei?) added a number of species (such as those in #8 ) prior to their books coming out. Which would sort of make them unofficial...? I'll look at it some time later, but I also recall the species being somewhat inconsistent as to when they're available. Either way, my opinion is that once an official version of a species is released, it should supersede versions based on NPC wording.

**Comment by bcholmes** [comment]
In a sense, it does supersede the other versions. The weird part happens when a person does not select the newer source: in that case, the older version is presented (often without talents).
